app/fields/components/shelf/hours.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App;
+
+use StoutLogic\AcfBuilder\FieldsBuilder;
+
+$fields = new FieldsBuilder('Live_Hours');
+
+//These are the fields for Live Hours a Shelf Exclusive extenstion for the CPT Branches
+
+$fields
```

Comment:
Let me know thoughts on this, I included in theme since it determines which "primary" branch shows in the header and allows the user some flexibility to override the hours (a common request)